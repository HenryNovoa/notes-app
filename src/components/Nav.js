export function Nav(){
    return (
        <nav>
            <div>Notes App</div>
        </nav>
    )
}