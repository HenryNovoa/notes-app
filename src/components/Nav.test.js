import { shallow } from 'enzyme'
import React from 'react'
import { Nav } from './Nav'
import { shallowToJson } from 'enzyme-to-json'


describe('Nav Component', ()=> {
    it('expect to render Nav component', ()=> {
        const navComponent = shallow(<Nav />)
        expect(shallowToJson(navComponent)).toMatchSnapshot()
    })
    
})
