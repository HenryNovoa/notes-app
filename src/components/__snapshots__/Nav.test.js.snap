// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component expect to render Nav component 1`] = `
<nav>
  <div>
    Notes App
  </div>
</nav>
`;
